src/main/java/com/epam/jwd/task01/entity/TariffTypes.java

```diff
@@ -0,0 +1,20 @@
+package com.epam.jwd.task01.entity;
+
+/**
+ * Enumeration to determine the possible types of tariffs
+ */
+
+public enum TariffTypes {
```

Comment:
Enums are classes. We call classes in the singular

Reply:
Agree